app/controllers/ops_controller/diagnostics.rb

```diff
@@ -965,9 +965,15 @@ def build_backup_schedule_options_for_select
   end
 
   def database_details
-    @database = MiqDbConfig.current.options
-    db_types = MiqDbConfig.get_db_types
-    @database[:display_name] = db_types[@database[:name]]
+    @database_details = Rails.configuration.database_configuration[Rails.env]
+    @database_display_name =
+      if @database_details["database"] != "vmdb_production"
+        _("External Database")
+      elsif @database_details["host"].in?([nil, "", "localhost", "127.0.0.1"])
```

Comment:
Yeah, I was thinking that when I moved it, but where would it live?  MiqDbConfig? :wink: :trollface: